cornice_swagger/swagger.py

```diff
@@ -126,9 +126,13 @@ def _build_paths(self, ignore=['head', 'options'], default_tags=None,
                         tags.append(root_tag)
 
                 # If method is defined for more than one ctype, get previous ones
-                ctypes = path.get(method_name.lower(), {}).get('consumes')
-                if ctypes and op.get('consumes') not in ctypes:
-                    op['consumes'] = ctypes + op['consumes']
+                ctypes = path.get(method_name.lower(), {}).get('consumes', [])
```

Comment:
We consult `path` because we might get multiple `view`\ s for the same `path`, which can e.g. dispatch according to the consumed content-type?